source/standalone/tutorials/03_envs/create_cartpole_base_env.py

```diff
@@ -153,7 +160,7 @@ def main():
             # step the environment
             obs, _ = env.step(joint_efforts)
             # print current orientation of pole
-            print("[Env 0]: Pole joint: ", obs["policy"][0][1].item())
+            # print("[Env 0]: Pole joint: ", obs["policy"][0][1].item())
```

Comment:
why is this one commented out now?

Reply:
clearing out the terminal while I was trying things out. Will revert before merge.